docs/image-verification.md

```diff
@@ -0,0 +1,23 @@
+# Image Verification
+
+To enable image verification, add a stanza like the following to the conainerd config:
+
+```yaml
+[plugins]
+  [plugins."io.containerd.image-verifier.v1.bindir"]
+    enabled = true
+    bin_dir = "/opt/containerd-image-verifier/bin"
+```
+
+Place one or more verifier binaries in `bin_dir` which conform to the following API:
```

Comment:
We should be explicit over the order in which binaries are invoked (which might be "no order" and we execute in parallel).